Source/WebCore/page/scrolling/ScrollAnchoringController.h

```diff
@@ -41,12 +41,18 @@ class ScrollAnchoringController final {
     void invalidateAnchorElement();
     void updateScrollPosition();
```

Comment:
I would call this `adjustScrollPositionForAnchoring()`

Reply:
Fixed.